client/src/components/admin/NewProduct.jsx

```diff
@@ -0,0 +1,178 @@
+import { useEffect, useState } from "react";
+import "../../styles/admin/newProduct.css";
+import { useSelector, useDispatch } from "react-redux";
+import { clearErrors, createProduct } from "../../actions/productAction";
+import { useAlert } from "react-alert";
+import { Button } from "@material-ui/core";
+import MetaData from "../layout/MetaData";
+import SideBar from "./Sidebar";
+import { useNavigate } from "react-router-dom";
+import { NEW_PRODUCT_RESET } from "../../constants/productConstants";
+
+const NewProduct = () => {
+  const dispatch = useDispatch();
+  const alert = useAlert();
+  const navigate = useNavigate();
+
+  const { loading, error, success } = useSelector((state) => state.newProduct);
+
+  const [name, setName] = useState("");
+  const [price, setPrice] = useState(0);
+  const [description, setDescription] = useState("");
+  const [category, setCategory] = useState("");
+  const [Stock, setStock] = useState(0);

```

Comment:
keep the consistency and convention. use camelCase or lowercase.

Reply:
resolved